python/nammu/view/EditSettingsView.py

```diff
@@ -264,6 +354,56 @@ def cancel(self, event=None):
         '''
         self.dispose()
 
+    def valiate_fontsize(self, input_size, target):
+        '''
+        Method to validate an input fontsize. The target parameter is used to
+        switch between validating the the console and the edit area. If the
+        value is invalid, return the previous user fontsize that was stored.
+        '''
+        if target == 'console':
+            target_key = 'console_style'
+        else:
+            target_key = 'edit_area_style'
```

Comment:
Can the target be already "console_stype" or "edit_area_style" from the place this is being called?

Reply:
The reason for this is so that we have a string containing `'console'` or `'edit area'` so that we can construct an error message using actual English rather than variable names. I will have a think about how to improve this.

Reply:
I have redesigned this so we pass in the relevant `target_key` and process it into plain English for the error message. See commit eb96dc085 for details.